rotkehlchen/tests/api/test_bitcoin.py

```diff
@@ -140,10 +117,11 @@ def test_add_delete_xpub(rotkehlchen_api_server):
     ), json=json_data)
     if async_query:
         task_id = assert_ok_async_response(response)
-        outcome = wait_for_async_task_with_result(rotkehlchen_api_server, task_id, timeout=180)
+        wait_for_async_task(rotkehlchen_api_server, task_id, timeout=180)
     else:
-        outcome = assert_proper_response_with_result(response)
-    _check_xpub_addition_outcome(outcome, xpub1)
+        assert_proper_response(response)
+
+    assert set(rotki.chain_manager.accounts.btc[2:]) == set(EXPECTED_XPUB_ADDESSES)
```

Comment:
Can you add also an assertion to verify that the balances were not queried?

Reply:
@yabirgb do you have any particular idea on how to check it? Also do you think we should check it in each account addition / removal test?

Reply:
No, not every asset. Just one to verify that the behavior is modified (it is) but also make sure we don't reintroduce it somehow. Doesn't `chain_manger.accounts.totals.assets` have them?

Reply:
Okay, added this check